### PR TITLE
Rename Asset Store to Rafter - example

### PR DESCRIPTION
**Pull request by mmitoraj**
**Description**

Changes proposed in this pull request:

- Renamed the Asset Store to Rafter

**Related issue(s)**
See also https://github.com/kyma-project/rafter/issues/10

### CODE REVIEW

**Comment by pPrecel** [comment]
Please update the main README.md

Reason:
https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_examples/149/pre-master-examples-governance/1192032711431163904

File: rafter/README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-# Asset Store
+# Rafter
```
Comment (pPrecel) [inline rafter/README.md]: Please update the `Installation` procedure  in point 4:
Change:
`kubectl get assets.assetstore.kyma-project.io webpage -o jsonpath='{.status.phase}'`
to:
`kubectl get assets.rafter.kyma-project.io webpage -o jsonpath='{.status.phase}'`

File: rafter/README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 ## Overview
 
-This example illustrates how to use the [Asset Store](https://kyma-project.io/docs/components/asset-store) to store static webpages.
+This example illustrates how to use [Rafter](https://kyma-project.io/docs/components/content-management) to store static webpages.
 
-By default, [Minio](https://min.io/) stores all resources on a cluster, but it also allows you to use different cloud providers. Read the Asset Store [tutorials](https://kyma-project.io/docs/components/asset-store#tutorials-tutorials) for more information.
+By default, [Minio](https://min.io/) stores all resources on a cluster, but it also allows you to use different cloud providers. Read Rafter [tutorials](https://kyma-project.io/docs/components/content-management#tutorials-tutorials) for more information.
```
Comment (kazydek) [inline rafter/README.md]:
```suggestion
By default, [Minio](https://min.io/) stores all resources on a cluster, but it also allows you to use different cloud providers. Read Rafter [tutorials](https://kyma-project.io/docs/components/rafter#tutorials-tutorials) for more information.
```

File: rafter/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-# Asset Store
+# Rafter
 
 ## Overview
 
-This example illustrates how to use the [Asset Store](https://kyma-project.io/docs/components/asset-store) to store static webpages.
+This example illustrates how to use [Rafter](https://kyma-project.io/docs/components/content-management) to store static webpages.
```
Comment (kazydek) [inline rafter/README.md]:
```suggestion
This example illustrates how to use [Rafter](https://kyma-project.io/docs/components/rafter) to store static webpages.
```

**Pull request merged**